invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/index.js

```diff
@@ -56,14 +57,28 @@ function renderRecordManagement(element) {
   );
 }
 
-if (recordVersionsAppDiv) {
-  ReactDOM.render(
-    <RecordVersionsList
-      record={JSON.parse(recordVersionsAppDiv.dataset.record)}
-      isPreview={JSON.parse(recordVersionsAppDiv.dataset.preview)}
-    />,
-    recordVersionsAppDiv
+if (recordVersionsAppDiv || recordEndorsementDisplayDiv) {
+  const record = recordVersionsAppDiv 
+    ? JSON.parse(recordVersionsAppDiv.dataset.record)
+    : JSON.parse(recordEndorsementDisplayDiv.dataset.record);
+
+  const VersionsWrapper = () => (
+    <VersionsProvider record={record}>
+      {recordVersionsAppDiv && (
+        <RecordVersionsList
+          record={record}
+          isPreview={JSON.parse(recordVersionsAppDiv.dataset.preview)}
+        />
+      )}
+      {recordEndorsementDisplayDiv && (
+        <EndorsementsDisplay record={record} />
+      )}
+    </VersionsProvider>
   );
+
+  if (recordVersionsAppDiv) {
+    ReactDOM.render(<VersionsWrapper />, recordVersionsAppDiv);
+  }
 }
```

Comment:
there are some problem in `if condition` 


VersionsWrapper will be created if `recordVersionsAppDiv || recordEndorsementDisplayDiv` and only render if recordVersionsAppDiv exist 

It create problem, why do we need to create `VersionsWrapper` when
recordVersionsAppDiv not exist 
recordEndorsementDisplayDiv exist 


it will also leave `recordEndorsementDisplayDiv` section empty

<img width="751" height="705" alt="image" src="https://github.com/user-attachments/assets/265b5d54-6fea-4531-867c-3d37774b0814" />